### ISSUE TITLE
html tags in i18n component

### ISSUE THREAD

**Issue by Demivan** [fluent-vue/fluent-vue#760]
### Discussed in https://github.com/Demivan/fluent-vue/discussions/759

<div type='discussions-op-text'>

<sup>Originally posted by **judge330** March 27, 2022</sup>
Hey!
Is there a way to use simple html tags in <i18n> component like with v-html?

Example:

Vue-Template:
```
<i18n path="general-register-info" tag="span">
    <template #showTermsModalSpan="{ generalPagesTerms }">
       <span class="underline cursor-pointer" @click.prevent="showModal('terms')">{{ generalPagesTerms }}</span>
    </template>
</i18n>
```

Imported ftl file with `<br>` and `<strong>`

```
general-pages-terms = Terms

general-register-info =
    Register here.<br> But respect <strong>our terms</strong> (see {$showTermsModalSpan}).
    .general-pages-terms = { general-pages-terms }
 ```

Result:
`Register here.<br> But respect <strong>our terms</strong> (see <ins>Terms</ins>).`

Expected:
Register here.
But respect **our terms** (see <ins>Terms</ins>).</div>

**Comment by Demivan** [comment]
It might be possible to implement with configurable markup parser: https://github.com/projectfluent/fluent.js/wiki/React-Overlays#custom-markup-parsers

**Comment by Demivan** [comment]
@judge330 I have finally implemented it in 3.2.0 release
